lib/widget/staggered_grid.dart

```diff
@@ -110,6 +112,13 @@ class EnsembleStaggeredGridState extends WidgetState<EnsembleStaggeredGrid>
   List<Widget> buildItems() {
     // children will be rendered before templated children
     List<Widget> children = [];
+    if (widget._controller.children != null && templatedChildren != null) {
+      throw LanguageError('Use either children or item-template');
```

Comment:
Use this string '${EnsembleStaggeredGrid.type} accepts either children or item-template', recovery: 'Please pick either children or item-template'

Reply:
Done @vusters